### ISSUE TITLE
Text_SendToAll doesn't exists in YSI 4.0?

### ISSUE THREAD

**Issue by knife-s** [pawn-lang/YSI-Includes#150]
Hello I am starting to use this awesome library with my gamemode to add multiple languages but I have a problem, I was following a tutorial about the y_text include in the sa-mp forums, apparently it is wrong or something.

It says what you must use: "ALL_PLAYERS" in the Text_Send function to send text for all players, but when I use it, it doesn't work (Doesn't show me/send me  the message) and any error appears in the server console (like text resource not found or something)

I was reading this issue https://github.com/Misiur/YSI-Includes/issues/66 (I made a comment on it) but apparently it is abandoned or something (no more replies)

So, I don't know if that bug still exists or if I am doing something wrong.

Note that if I use Text_Send with "playerid" works very well.

In addition, I am planning to simulate ALL_PLAYERS with a foreach loop, like this:

foreach(new iplayerid : Player)
{
   Text_Send(iplayerid,$USER_HAS_CONNECTED, PName(playerid), playerid, GetPlayerCountry(playerid));
}

but I don't know if it is OK (thinkin in speed/optimizations)

Thank you so much

**Comment by Y-Less** [comment]
I've been neglecting y_text bugs for a long time because I started trying to rewrite it in a more modular testable fashion.  However, that's just not working - the more I do, the more I realise the old version did and the more corner-cases it handled quite well.  The code might be messy, but it worked well for the most part other than a few issues with colours.  I'm just going to abandon efforts to rewrite it (since a ground-up rewrite is rarely the best solution anyway) and work on fixing the original.  I've also got some ideas for performance improvements but they're just modifications to existing code, not starting from scratch.

Point being, I'll try get to these now.
